packages/clay-charts/demos/index.html

```diff
@@ -338,8 +338,41 @@ <h2>Bubble</h2>
 			]
 		};
 
-		var lineChart = new metal.LineChart({
-			columns: dummyData.columns_1,
+		var lineChartData = {
+			columns_1: 'data/dummy-data-columns-1.json',
+			columns_2: 'data/dummy-data-columns-2.json',
+			columns_3: function() {
+				return Promise.resolve([
+					{
+						data: [230, 190, 300, 500, 300, 400],
+						id: 'data1',
+						name: 'Data 1 (this data set has 3 columns)'
+					},
+					{
+						data: [50, 20, 10, 40, 15, 25],
+						id: 'data2'
+					},
+					{
+						data: [130, 150, 200, 300, 200, 100],
+						id: 'data3'
+					}
+				])
+			},
+			columns_4: [
+				{
+					data: [50, 20, 10, 40, 15, 25],
+					id: 'data1',
+					name: 'Data 1'
+				},
+				{
+					data: [130, 150, 200, 300, 200, 100],
+					id: 'data2'
+				}
+			]
+		};
+
+		var lineChartWithData = new metal.LineChart({
+			columns: lineChartData.columns_1,
```

Comment:
still columns ;)

Reply:
Yes, weren't we still supposed to support columns as well as data?

Reply:
Yeah, but no need to showcase it in the demos :)

Reply:
Make sense, ok thanks